src/ResourceManager/DataLakeStore/Commands.DataLakeStore.Test/Commands.DataLakeStore.Test.Netcore.csproj

```diff
@@ -24,7 +24,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.DataLake.Store" Version="2.3.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.DataLake.Store" Version="2.4.1-preview" />
```

Comment:
@lewu-msft be sure to also bump this version in `Commands.DataLakeStore.Netcore.csproj` -- this is why the Travis CI job is currently failing